gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarColorScheme.kt

```diff
@@ -0,0 +1,7 @@
+package com.gravatar.quickeditor.ui.editor
+
+public enum class GravatarColorScheme {
```

Comment:
I'm open about the naming, so let me know if you think there's a better one.

Reply:
⚠️ I don't like `ColorScheme` because it can be easily confused with [material/ColorScheme](https://developer.android.com/reference/kotlin/androidx/compose/material3/ColorScheme).

What about:

```suggestion
public enum class GravatarUiMode {
```

Reply:
Makes sense! I will update it.

Reply:
Done.